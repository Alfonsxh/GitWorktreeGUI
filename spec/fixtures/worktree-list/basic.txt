worktree /Users/project/main
HEAD abc1234567890abcdef1234567890abcdef12345
branch refs/heads/main

worktree /Users/project/feature-ai
HEAD def1234567890abcdef1234567890abcdef12345
branch refs/heads/feature-ai

worktree /Users/project/bugfix-123
HEAD 1234567890abcdef1234567890abcdef12345678
branch refs/heads/bugfix-123