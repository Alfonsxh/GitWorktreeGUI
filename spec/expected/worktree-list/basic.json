[
  {
    "path": "/Users/project/main",
    "head": "abc1234567890abcdef1234567890abcdef12345",
    "branch": "main",
    "locked": false,
    "isMainWorktree": true
  },
  {
    "path": "/Users/project/feature-ai",
    "head": "def1234567890abcdef1234567890abcdef12345",
    "branch": "feature-ai",
    "locked": false,
    "isMainWorktree": false
  },
  {
    "path": "/Users/project/bugfix-123",
    "head": "1234567890abcdef1234567890abcdef12345678",
    "branch": "bugfix-123",
    "locked": false,
    "isMainWorktree": false
  }
]